app/views/adjustments/new.html.erb

```diff
@@ -9,7 +9,7 @@
       <i class="fa fa-dashboard"></i> Home
     <% end %>
     </li>
-    <li><%= link_to "Distributions", (adjustments_path) %></li>  
+    <li><%= link_to "Distributions", (adjustments_path) %></li>
```

Comment:
If we are linking to the `adjustments_path`, should it be a link to *Inventory Adjustments* or something like that?

Reply:
This was fixed in the other PR -- you are correct though.